test/cypress/integration/05-user_v2.test.js

```diff
@@ -1,26 +1,96 @@
 describe('New users profiles', () => {
+  before(() => {
+    cy.visit('/xdamman/v2');
+  });
   describe('Contributions section', () => {
-    it.skip('Shows contributions with date since and amount contributed', () => {
-      // TODO
+    it('Shows contributions with date since and amount contributed', () => {
+      cy.get('[data-cy=section-contributions]').click({ force: true });
+      cy.hash().should('eq', '#section-contributions');
+      cy.get('[data-cy=section-contributions-title]').contains('Contributions');
+      cy.get('[data-cy=contribution-date-since]')
+        .first()
+        .contains('Collective Admin since');
+      cy.get('[data-cy=contribution-date-since]')
+        .first()
+        .contains('August 2016');
+      cy.get('[data-cy=amount-contributed]')
+        .first()
+        .contains('amount contributed');
+      cy.get('[data-cy=amount-contributed]')
+        .first()
+        .contains('€267');
     });
 
     it.skip('Can load more', () => {
       // TODO
     });
 
-    it.skip('Can filter by contribution type (admin, financial...etc)', () => {
-      // TODO
+    it('Can filter by contribution type (admin, financial...etc)', () => {
+      cy.get('[data-cy=filters]');
+      cy.get('[data-cy="filter-button core"]').click({ force: true });
+      cy.get('[data-cy=collective-contribution]')
+        .first()
+        .get('[data-cy=caption]')
+        .contains('Collective Admin since');
+      cy.get('[data-cy="filter-button financial"]').click({ force: true });
```

Comment:
Rather than changing how the filters work it tests, you could have copied the behavior of the existing tests and follow this pattern, which I personally find cleaner:

```suggestion
      cy.contains('[data-cy="filter-button"]', 'Financial').click();
```

Reply:
through `[data-cy="filter-button financial"]` it feels more declarative to me such that - filter-button which also has a property called - financial

Reply:
I don't really understand this sentence. Do you mean declarative as in declarative programming (opposed to imperative programming)?

Reply:
> a formal or explicit statement or announcement

This - english meaning for it

Reply:
Okay!
by picking up `data-cy="filter-button financial"` I more mean by picking a component which has both the values for `data-cy` attr. Not picking/selecting up a component which just has a value for - `filter-button` and then checking the content - which eventually fails if you simply do this - `cy.get('[data-cy="filter-button"])` as `filter-button` is just not only the value and cypress couldn't find it out

Reply:
Thanks for the clarifications. I still think that it's nice to keep the filter value out of the property (it's a valid use case to check for the presence of a filter button without considering the value) but that's personal preference, so no blocker here.